### PR TITLE
Update issue_template.md

### PR DESCRIPTION
**Pull request by 55Honey**
Shorten, improve readability for newbies.

The feedback on our issue template has repeatedly been to be "confusing". I dared to shorten the instructions.

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -24,7 +20,7 @@ ________________________________________________________________________________
 <!--
 ________________________________________________________________________________________________________________________________________
 ________________________________________________________________________________________________________________________________________
-Specify the content phase where this bug belongs to, for example "1-19" or "20-29", etc...
+2) Specify the content phase where this bug belongs to, for example "1-19" or "20-29", etc...
```
Comment (illfated) [inline issue_template.md:23]: Hope you don't mind a suggestion for a minor sentence structure improvement.
```suggestion
2) Specify the content phase this bug belongs to, for example "1-19" or "20-29", etc...
```

File: issue_template.md [hunk 2]
```diff
@@ -36,19 +32,11 @@ ________________________________________________________________________________
 <!--
 ________________________________________________________________________________________________________________________________________
 ________________________________________________________________________________________________________________________________________
-Add a bug description line, then fill in the required fields for your problem (based on the needs of your problem)
-Delete the "<!--" symbols at the beginning and at the end according to the field you need, the fields you don't need to fill ignore them.
+3a) Add a small description.
+3b) Add links to point to the quest/NPCs/spells/items/... related to your problem.
+Delete the "<!--" symbols at the beginning and at the end according to the field you need, you can ignore the others.
```
Comment (illfated) [inline issue_template.md:37]: Suggestion (not very important, feel free to ignore):
```suggestion
Delete the "<!--" symbols at the beginning and at the end according to the field you need, please ignore the others.
```

File: issue_template.md [hunk 3]
```diff
@@ -3,17 +3,13 @@
...
-Specify to which type of Faction the problem in question belongs
-(and I don't mean which Character you encountered the problem with if Horde or Alliance, I mean if the Problem is specific to the faction),
-For example:
-There may be a quest that is the same for both Horde that for Alliance, but that maybe that of Horde works and that Alliance doesn't,
-so indicate to which faction the problem belongs through these 2 Images, you need to delete ONLY the relates symbols:
-from the line <!- -> leaving the link of of interest in this way:
-" ![image](https://user-images.githubusercontent.com/1884642/108204869-3a88d100-711c-11eb-8179-e1b9b73ed450.png) " (Without quotes)
+Guide to issues:
+ - Text in between <!-- and --> is not visible. It serves to guide you through the blueprint. Leave it as is.
+1) Specify to which type of Faction the problem in question belongs. If the issue can happen on only one faction, remove the arrows before and
```
Comment (illfated) [inline issue_template.md:8]: Very minor grammar detail:
```suggestion
1) Specify to which type of Faction the problem in question belongs. If the issue can happen to only one faction, remove the arrows before and
```

**Review by illfated — APPROVED** [review]
Looks like a good improvement overall, good idea to shorten the text for readability.

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
thanks @55Honey and @illfated !

File: issue_template.md (hunk 1 above)
Comment (55Honey) [reply issue_template.md:23]: I sure dont mind a helpful hand, thx!

Comment (illfated) [reply issue_template.md:23]: Looks like this suggestion got skipped, maybe I should have added a secondary text line underneath it to highlight that there are (at least) 2 ways of phrasing this sentence:
```
2) Specify the content phase this bug belongs to, for example "1-19" or "20-29", etc.
2) Specify the content phase where this bug belongs, for example "1-19" or "20-29", etc.
```

**Pull request merged**